### PR TITLE
feat: sort results table columns by show_underlying_values

### PR DESCRIPTION
**Pull request by rephus**
Closes: https://github.com/lightdash/lightdash/issues/3545

### Description:

Screenshot from 2022-10-24 10-04-08

### CODE REVIEW

File: packages/frontend/src/components/UnderlyingData/UnderlyingDataResultsTable.tsx [hunk 1]
```diff
@@ -19,7 +20,17 @@ const UnderlyingDataResultsTable: FC<{
...
+    sortByUnderlyingValues: (
+        columnA: TableColumn,
+        columnB: TableColumn,
+    ) => number;
+}> = ({
+    fieldsMap,
+    resultsData,
+    isLoading,
+    hasJoins,
+    sortByUnderlyingValues,
```
Comment (ZeRego) [inline packages/frontend/src/components/UnderlyingData/UnderlyingDataResultsTable.tsx:32]: Should this function be called somewhere in this component ?

**Review by TuringLovesDeathMetal — APPROVED** [review]
Nice! this looks good to me :) 

Just to double check - what happens if there's no `show_underlying_values` .yml applied?

File: packages/frontend/src/components/UnderlyingData/UnderlyingDataResultsTable.tsx (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/components/UnderlyingData/UnderlyingDataResultsTable.tsx:32]: yes of course, I reverted the wrong line...

**Comment by rephus** [comment]
> Just to double check - what happens if there's no `show_underlying_values` .yml applied?

Then we should the default dimensions with the default sorting

**Review by ZeRego — APPROVED** [review]

**Pull request merged**